inc/patterns/general-404.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * 404 content.
+ */
+return array(
+	'title'      => __( '404 content', 'twentytwentytwo' ),
+	'categories' => array( 'twentytwentytwo-general' ),
+	'content'    => '<!-- wp:heading {"style":{"typography":{"fontSize":"clamp(4rem, 40vw, 20rem)","fontWeight":"200","lineHeight":"1","spacing":{"margin":{"top":"0","bottom":"0"}}}},"className":"has-text-align-center"} -->
+					<h2 class="has-text-align-center" style="font-size:clamp(4rem, 40vw, 20rem);font-weight:200;line-height:1;margin-top:0;margin-bottom:0;">404</h2>
+					<!-- /wp:heading -->
+					<!-- wp:paragraph {"align":"center"} -->
+					<p class="has-text-align-center">' . esc_html( 'This page could not be found. Maybe try a search?', 'twentytwentytwo' ) . '</p>
```

Comment:
Implemented the suggestion from here https://github.com/WordPress/twentytwentytwo/pull/154#issuecomment-951122568